backend/buildeR.DAL/Migrations/20200806180242_add-migration UpdateMigration.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace buildeR.DAL.Migrations
+{
+    public partial class addmigrationUpdateMigration : Migration

```

Comment:
Select understandable names for migrations in future

Reply:
this migration was added only because previous changes wasn't applied to db and data wasn't able to work with queries.
But I got you)